core/src/main/java/hudson/model/StringParameterValue.java

```diff
@@ -70,7 +70,7 @@ public String resolve(String name) {
     }
 
     @Override
-    public Object getValue() {
+    public String getValue() {
```

Comment:
I thought this would be binary compatable, but I could not find anything concrete in [https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html](https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html)

Reply:
Write a test then?

Reply:
I'm going to do a manual test.  Writing one would invlove checking in classes and hacking surefire etc...

Reply:
javac should inject a bridge method with the Object return type for you by virtue of being an overload

Reply:
java does indeed inject the bridge method.

`javap -cp classes -v hudson.model.StringParameterValue`
produces: 
```
  public java.lang.Object getValue();
    descriptor: ()Ljava/lang/Object;
    flags: ACC_PUBLIC, ACC_BRIDGE, ACC_SYNTHETIC
    Code:
      stack=1, locals=1, args_size=1
         0: aload_0
         1: invokevirtual #26                 // Method getValue:()Ljava/lang/String;
         4: areturn
      LineNumberTable:
        line 39: 0
      LocalVariableTable:
        Start  Length  Slot  Name   Signature
            0       5     0  this   Lhudson/model/StringParameterValue;
```

Reply:
@stephenc so I gather you'd approve this change?